CMakeLists.txt

```diff
@@ -81,8 +81,10 @@ if(NOT USE_PREBUILT_LLVM)
 
     if(NOT LLVM_EXTERNAL_CLANG_SOURCE_DIR)
         set(CLANG_SOURCE_DIR ${LLVM_SOURCE_DIR}/tools/clang)
+        set(CLANG_BASE_REVISION release_11)
     elseif(EXISTS "${LLVM_EXTERNAL_CLANG_SOURCE_DIR}/CMakeLists.txt")
         set(CLANG_SOURCE_DIR "${LLVM_EXTERNAL_CLANG_SOURCE_DIR}")
+        set(CLANG_BASE_REVISION release/10.x)
```

Comment:
why use different format as previews "release_11" ?

Reply:
This is align to ocl-open-100 branch. We should use same revision but I'm not sure what's usage for second path. I'll check and update it later.